README.md

```diff
@@ -9,23 +9,69 @@
 
 ---
 
-Crane (FinOps Crane) is a cloud native open source project which manages cloud resources on Kubernetes stack, it is inspired by FinOps concepts.
+## What is Crane
 
-## Introduction
+> English | [中文](README_zh.md)
 
-The goal of Crane is to provide a one-stop-shop project to help Kubernetes users to save cloud resource usage with a rich set of functionalities:
+Crane is a full lifecycle FinOps Platform for Cloud Resource Analytics and Economics in Kubernetes clusters.The goal is not only help user to manage cloud cost easier but also ensure the quality of applications.
```

Comment:
Crane is a FinOps Platform for Cloud Resource Analytics and Economics in Kubernetes clusters.

Reply:
missing a space after `.`